Gruntfile.js

```diff
@@ -177,6 +177,7 @@ module.exports = function (grunt) {
                 devLayout = grunt.file.read('./test/e2e/layouts/cdn-dev.tpl', {encoding: 'utf8'}),
                 indexLayout = grunt.file.read('./test/e2e/layouts/index.tpl', {encoding: 'utf8'}),
                 npmLayout = grunt.file.read('./test/e2e/layouts/npm.tpl', {encoding: 'utf8'}),
+                sharedFunctions = grunt.file.read('./test/e2e/partials/shared-functions.js', {encoding: 'utf8'}),
```

Comment:
why not just copy to `target/js` and loads from template directly?

Reply:
With a `<script>` tag? That could work. In that case I would probably want a different folder name than `partials` if it's not being used in the templating system.

Reply:
yeah, since this is just shared utilities, we just just load from base tpl.